dockerfiles/web/Dockerfile

```diff
@@ -30,8 +30,9 @@ USER django:django
 WORKDIR /opt/poetry
 COPY pyproject.toml /opt/poetry
 COPY poetry.lock /opt/poetry
+RUN poetry config --local virtualenvs.in-project true
```

Comment:
Does the `--local` argument modify `pyproject.toml`? Could we add the configuration there instead. Would then need to add the `.venv` directory to `.gitignore` if it is not there already.

Reply:
Apparently it needs to by in a separate `poetry.toml` file: https://github.com/python-poetry/poetry/pull/1272, which I have added now.